enterprise_gateway/services/processproxies/container.py

```diff
@@ -212,6 +231,33 @@ async def confirm_remote_startup(self) -> None:
                         self.pgid = 0
             else:
                 self.detect_launch_failure()
+        self.kernel_events_to_occurrence_time = {}
+
+    def _handle_pending_kernel(self):
+        self.log.debug("Sampling kernel container events")
+        kernel_pod_events = self.get_container_events()
+        for event in kernel_pod_events:
+            if (
+                event.type in self.kernel_launch_terminate_on_events
```

Comment:
Since the event type and reason need to be matches, is there a doc reference we could add that enumerates the various values that could be expected?  That said, I suppose folks will need to be reactive (vs. proactive) in this sense and will have the event information when troubleshooting, but it still might be helpful to have a link or two, or add the enumeration directly.

Reply:
Added a docstring to `handle_pending_kernel` method in `container.py`, as well as a more detailed configuration help. I found it hard to find a detailed list of Kubernetes events by type and reason, so I added an enumeration which may differ in different platforms (or even versions of K8S itself). I agree that folks will need to be reactive, but I did also add a link to a guide I found helpful with finding the correct event type and reason.